packages/fluxible-router/docs/api/HistoryWithHash.md

```diff
@@ -27,7 +27,7 @@ By default, the hash fragments are just url paths.  With `HistoryWithHash`, you
 
 ```js
 var handleHistory = require('fluxible-router').handleHistory;
```

Comment:
should we change these to import?

Reply:
I've updated the examples to use modern es ;)